Cargo.toml

```diff
@@ -35,21 +36,17 @@ humantime = "2.1.0"
 hyper = { version = "0.14.23", default-features = false }
 log = "0.4.17"
 metrics = { version = "0.20.1", default-features = false }
-postcard = { version = "1.0.2", default-features = false }
+postcard = { version = "1.0.8", default-features = false }
 rand = "0.8.5"
-reqwest = "0.11.13"
+reqwest = { version = "0.11.13" }
```

Comment:
Minor point but the request and tokio dependencies here now use the expanded (brace) syntax but are only specifying version while the other dependencies in the file which only specify version use the shorthand syntax. It doesn't really matter, it's just a bit inconsistent :)